### PR TITLE
Support for accessibility label and hint

### PR DESCRIPTION
**Pull request by goderbauer**
Required for https://github.com/flutter/flutter/issues/12271.

### CODE REVIEW

File: shell/platform/android/io/flutter/view/AccessibilityBridge.java [hunk 1]
```diff
@@ -615,5 +620,21 @@ private float min(float a, float b, float c, float d) {
...
             return Math.max(a, Math.max(b, Math.max(c, d)));
         }
+
+        private String getValueLabelHint() {
+            StringBuilder sb = new StringBuilder();
+            boolean first = true;
+            String[] array = { value, label, hint };
+            for (String word: array) {
+                if (word != null && (word = word.trim()).length() > 0) {
+                    if (first)
```
Comment (jason-simmons) [inline shell/platform/android/io/flutter/view/AccessibilityBridge.java]: this can be changed to:
  if (sb.length() > 0) sb.append(', ');

Comment (aam) [reply shell/platform/android/io/flutter/view/AccessibilityBridge.java]: I would imagine `if (!sb.isEmpty())` is better than `if (sb.length() > 0)` still.

Comment (goderbauer) [reply shell/platform/android/io/flutter/view/AccessibilityBridge.java]: Looks like StringBuilder doesn't have an isEmpty method: https://docs.oracle.com/javase/7/docs/api/java/lang/StringBuilder.html

Comment (aam) [reply shell/platform/android/io/flutter/view/AccessibilityBridge.java]: Ah, got you.

**Pull request merged**